lib/region_config.js

```diff
@@ -58,6 +58,9 @@ function configureEndpoint(service) {
 
       // set global endpoint
       service.isGlobalEndpoint = !!config.globalEndpoint;
+      if (config.signatureRegion) {
```

Comment:
We use `signingRegion` in other SDKs like [java SDK](https://github.com/aws/aws-sdk-java-v2/blob/cdfb7d3f6a7dafb37be84f6e95e49feb0d0af273/core/auth/src/main/java/software/amazon/awssdk/auth/signer/params/Aws4SignerParams.java#L44). It more common to use that name.

Reply:
updated